### DISCUSSION TITLE
How to resolve :- requests.exceptions.SSLError: HTTPSConnectionPool(host='cloud.getdbt.com', port=443): Max retries exceeded with url: /api/v2/accounts/56461/jobs/84200/run/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))

### DISCUSSION THREAD

**Discussion by rajans163** [apache/airflow#24396] [Q&A]
I am stuck with SSl error for long while accessing dbt_cloud with Airflow

requests.exceptions.SSLError: HTTPSConnectionPool(host='cloud.getdbt.com', port=443): Max retries exceeded with url: /api/v2/accounts/56461/jobs/84200/run/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)'))).
![airflow error with dbt_cloud](https://user-images.githubusercontent.com/61928013/173208174-548be3ec-0cac-4096-bd25-647cbe2ea0ad.JPG)

No No idea how to resolve it

**Comment by potiuk** [comment]
It's not specific to Airflow - it's the question of your deployment/environment where airflow is installed, version of the OS, version of libraries - some of them are just likely very old. It's generally a good idea to upgrade your dependencies often.

Simply, for whatever reason you use old certificats/old SSL libraries and they do not have newer public certificates installed to verify the dbt private certificate.

You have to look through the internet for similar errors and apply remedies found there. An example here (and one of the remedies if you look closer was upgrading pyopenssl library)  https://github.com/psf/requests/issues/4246 - but there are many othe reasons it might happen and various solutions might apply.

Generally installing and upgrading to latest "good versions" of libraries should help - both on the "base installation/operating system" and libraries used by Airlfow. Maybe even upgrading Airflow itself (bacuase newer airflow uses newer dependencies). And When installing/upgrading airflow you should use constraints as described here - https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html then at least python dependencies will be upgraded to newer versions (but if your problem is with OS libraries, they might need separate actions).
